### DISCUSSION TITLE
Adding working tutorials?

### DISCUSSION THREAD

**Discussion by ivicajan** [noaa-ocs-modeling/OCSMesh#108] [Q&A] [answered]
Hi,
I am starting my meshing project using OCSMesh, and having some working tutorials would speed up my efforts.

Thanks in advance,
Ivica

p.s. I'm using jupyter notebook and at the end could provide it as a example

**Comment by SorooshMani-NOAA** [comment, marked as answer]
Hi @ivicajan thanks for your offer. Currently we decided to put all relevant tutorials in a separate repository. You can take a look at 
- https://github.com/SorooshMani-NOAA/OCSMeshTutorial/tree/main/UFS_Nov_2022 
- https://github.com/SorooshMani-NOAA/OCSMeshTutorial/blob/main/Tutorials_2023/golfo_san_jose.ipynb

for a OCSMesh introduction tutorial. If you have more advanced questions, I'd be happy to answer. Also please let me know if you run into any issues setting up your environment. One piece of information currently missing from all the tutorials and readmes is that now you can install OCSMesh directly from conda by `conda install -cconda-forge ocsmesh`. As far as conda installations are concerned, I'd prefer `mamba`, since it solves the environment much faster than `conda`!

**Reply by ivicajan** [reply]
Great, 
thanks a lot for tutorials and swift response. Will let you know if stuck.

Cheers,
Ivica

**Comment by SorooshMani-NOAA** [comment]
@bob113313 I saw a notification email about a trouble you faced using OCSMesh, is it resolved? I don't see your comment here when I clicked on the notification link!
